.github/workflows/ci.yaml

```diff
@@ -50,5 +51,52 @@ jobs:
           docker pull gcr.io/oak-ci/oak:latest
           docker build --pull --cache-from=gcr.io/oak-ci/oak:latest --tag=gcr.io/oak-ci/oak:latest .
 
+      # Init .git repository used by check_generated.
+      # Workaround for https://github.com/GoogleCloudPlatform/cloud-builders/issues/236
```

Comment:
So is this also an issue for GH Actions?

Reply:
Deleted